src/__tests__/index-test.tsx

```diff
@@ -265,6 +270,27 @@ describe('redux-react-hook', () => {
       expect(renderCount).toBe(2);
     });
 
+    it('renders once if have multiple useMappedState', () => {
+      let renderCount = 0;
+      const Component = ({ prop }: { prop: any }) => {
+        const mapState1 = React.useCallback((s: IState) => s.bar, [prop]);
+        const mapState2 = React.useCallback((s: IState) => s.foo, [prop]);
+        useMappedState(mapState1);
+        useMappedState(mapState2);
+        renderCount++;
+        return null;
+      };
+
+      render(<Component prop={1} />);
+
+
+      updateStoreWithoutAct({ bar: 11, foo: '11' })
+      updateStoreWithoutAct({ bar: 12, foo: '12' })
+      act(()=>{})
```

Comment:
please explain

Reply:
just flushing all hooks before checking for rerenders counts. Not sure if it's needed